### DISCUSSION TITLE
Migration issues with 0.11

### DISCUSSION THREAD

**Discussion by Jolaxs** [histoire-dev/histoire#333] [Q&A]
I tried upgrading histoire from 0.10.7 to 0.11.5 but now have the following error when histoire collects stories
`Error: [vite-node] Failed to load virtual:$histoire-generated-globalThis-setup`
This happens for every story. 
I did not see any migration guide so if someone knows what may be the issue it would be appreciated.

**Comment by Akryum** [comment]
You probably have a dependency that doesn't play well with NodeJS. Try putting your setup file under an object with only `browser`:

```setupFile: { browser: '...' }```
